app/kafka/kafka.go

```diff
@@ -0,0 +1,22 @@
+package kafka
+
+import (
+	"fmt"
+
+	kafka "github.com/segmentio/kafka-go"
+)
+
+func newKafkaWriter(kafkaURL, topic string) *kafka.Writer {
+	return &kafka.Writer{
+		Addr:     kafka.TCP(kafkaURL),
+		Topic:    topic,
+		Balancer: &kafka.LeastBytes{},
+	}
+}
+
+func Connect() *kafka.Writer {
+	_writer := newKafkaWriter("localhost:29092", "events")
```

Comment:
Read these parameters from env vars. With this events topic, we will have multiple topics for example `souq-mint`, `souq-ask` and so on.